### ISSUE TITLE
Any plan for a larger context size?

### ISSUE THREAD

**Issue by HermitSun** [nlpxucan/WizardLM#109]
A larger context size is likely to be helpful for many downstream tasks.

Recently proposed researches has extended LLaMA models' context size to 8k and beyond via RoPE scaling. 

Do you have any plan for the context size extension?

**Comment by edwardzjl** [comment]
Absolutely, particularly when considering the remarkable performance of the 70B model. Expanding the context length would undoubtedly unlock its full potential, allowing it to excel in tasks such as COT or document retrieval. The ability to capture and comprehend extensive contextual information is a pivotal aspect, and enhancing this capacity would significantly elevate the model's capabilities in a variety of applications.
